lib/utils.ex

```diff
@@ -0,0 +1,15 @@
+defmodule MLLP.Utils do
```

Comment:
I think we might want to get rid of this wrapper and keep the Logger calls as close to the call site as possible so that that information captured by Logger doesn't start here, maybe not a huge deal though either. 

Other part, we want the prefix to be contextual. Users can already configure the logger to emit log lines with the application name (i.e., `mllp`). However, even if they do, there some context lost (i.e., did a error come from the client or receiver?). 

So, perhaps we prefix with MLLP.Client or MLLP.Receiver, thoughts?

Reply:
Agreed, having a hardcoded prefix for everything is not good.
[019cba9](https://github.com/HCA-Healthcare/elixir-mllp/pull/74/commits/019cba99fd0c8ded004333bdf1504d9e9caac9a4) tries to address it by passing an optional prefix to `MLLP.Utils.log/3`.

Reply:
This looks good to me, if you decide to go down this route, but also note that Logger includes both module, MFA, and application in its metadata. So if you include the application in the metadata, you would see `application=mllp` in all relevant log entries.

Reply:
Right, but we'll need to ditch the wrapper I do believe, as using it will negate that option. But I hear you, maybe we just don't prefix anything and leave it to the user to configure this based on your suggestion. Still we can use this PR to improve upon the messages themselves (also need to refresh myself on the state of structured logging in elixir now).